homeassistant/components/tplink/light.py

```diff
@@ -145,8 +145,8 @@ async def async_setup_entry(
                 TPLinkSmartLightStrip(
```

Comment:
Changes to light platform probably belong to the light/fan PR? Assuming this is not a must have to avoid breaking something.

Reply:
This was just to bring it in line with the kwargs only `__init__` changes now in the `tplink_rewrite_sensor` branch.